jwst/datamodels/fits_support.py

```diff
@@ -377,6 +377,7 @@ def _save_history(hdulist, tree):
         history = tree['history']
     else:
         history = tree['history'].get('entries', [])
+        tree['history'] = history
```

Comment:
`AsdfFile.tree.history` is now a dictionalry, not a list. Why is it necessary to assign a list to `.tree`? 
Is there an example that failed before this change?